lib/geordi/chromedriver_updater.rb

```diff
@@ -81,8 +87,13 @@ def fetch_response(url, error_message)
       if response.is_a?(Net::HTTPSuccess)
         yield(response)
       else
-        Interaction.fail(error_message)
+        raise ProcessingError, error_message
       end
+
+    # Rescue Errno::NOERROR, Errno::ENOENT, Errno::EACCES, Errno::EFAULT, Errno::ECONNREFUSED, Errno::ECONNABORTED, Errno::ECONNRESET, Errno::EHOSTDOWN, Errno::EHOSTUNREACH, ...,
+    # all of which are a subclass of SystemCallError
+    rescue SystemCallError => e
+      raise ProcessingError, "Could not connect to Server: #{e.message}"
```

Comment:
Nice rescue :muscle: 

I think we'd need to be a little more abstract here though:
```suggestion
      raise ProcessingError, "Request failed: #{e.message}"
```